### ISSUE TITLE
Need to add some more values to the hardcoded list of strange CS terms

### ISSUE THREAD

**Issue by JohnAllenTech** [codu-code/codu#807]
## Context

Currently we arent uppercasing TypeScript or JS

This is just a case of adding them to 

```
const commonCamelCaseCSWords = new Map([
  ["javascript", "JavaScript"],
  ["css", "CSS"],
]);
```

## Expected Behavior

<img width="349" alt="image" src="https://github.com/codu-code/codu/assets/46611809/e55bac0e-6c15-4360-ad30-f47a60d2b8d3">

## Current Behavior
<img width="349" alt="image" src="https://github.com/codu-code/codu/assets/46611809/067fd2d4-02b6-40d8-93fe-8a3b42202382">

## Screenshots
- Shown above
## Steps to reproduce

Open up codu.co and look at the popular tags

## Additional info

Provide any additional information here

**Comment by the-Tiege** [comment]
I would like to try to resolve this issue.

**Comment by JohnAllenTech** [comment]
@NiallJoeMaher can probably add @the-Tiege to this before closing it so he gets the credit

**Comment by JohnAllenTech** [comment]
Solved in #808

**Comment by JohnAllenTech** [comment]
Solved in #808
